scss/layout/_scaffolding.scss

```diff
@@ -0,0 +1,122 @@
+@import '../abstracts/variables';
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+}
+
+[class*='container'] {
+  margin: 0 auto;
+  padding-right: $column-gutter;
+  padding-left: $column-gutter;
+}
+.container-small {
+  max-width: $device-sm;
+}
+.container {
+  position: relative;
+  max-width: 66rem;
+}
+.container-large {
+  position: relative;
+  max-width: $device-lg;
+}
+
+.row {
+  display: flex;
+  flex-flow: row wrap;
+  margin-right: -($column-gutter/2);
+  margin-left: -($column-gutter/2);
+  &.align-middle {
+    align-items: center;
+  }
+  &.align-bottom {
+    align-items: flex-end;
+  }
+  &.align-stretch {
+    align-items: stretch;
+  }
+  &.align-left {
+    justify-content: flex-start;
+  }
+  &.align-center {
+    justify-content: center;
+  }
+  &.align-right {
+    justify-content: flex-end;
+  }
+  &.align-between {
+    justify-content: space-between;
+  }
+  &.reverse {
+    flex-wrap: wrap-reverse;
+  }
+}
+
+[class*='col'] {
+  flex-basis: 100%;
+  padding: 0 $column-gutter/2;
+}
+
+@mixin col-size($col) {
+  flex-basis: (100% * $col / 12);
+  max-width: (100% * $col / 12);
+}
+
+@each $col in (1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12) {
+  .col-xs-#{$col} {
+    @include col-size($col);
+  }
+}
+
+@media (min-width: $device-sm) {
+  html,
+  body {
+    font-size: 14px;
+  }
+  @each $col in (1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12) {
+    .col-sm-#{$col} {
+      @include col-size($col);
+    }
+  }
+  .col-sm-auto {
+    flex-basis: auto;
+  }
+}
+
+@media (min-width: $device-md) {
+  html,
+  body {
+    font-size: 15px;
+  }
```

Comment:
dittO: we probably don't want this font size to propagate lest we break webb3? Also font changes like this should probably stay in typography.